android/app/src/main/java/com/example/testredirect/MainActivity.kt

```diff
@@ -43,14 +42,21 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun loadCustomTab(url: String) {
-        println("Loading Custom Tab")
-        val builder = CustomTabsIntent.Builder()
-        val customTabsIntent = builder.build()
-        customTabsIntent.launchUrl(this, Uri.parse(url))
+        try {
+            // Attempts to start custom tab with Google Chrome
+            val builder = CustomTabsIntent.Builder()
+            val customTabsIntent = builder.build()
+            customTabsIntent.intent.setPackage("com.android.chrome")
+            customTabsIntent.launchUrl(this, Uri.parse(url))
```

Comment:
[Suggestion] Maybe add some comments that this is only recommended if it's a for a GBP Payment where the vast majority of providers have support for App2App